article-analyzer/src/main/java/co/inventorsoft/academy/spring/models/NotificationType.java

```diff
@@ -0,0 +1,8 @@
+package co.inventorsoft.academy.spring.models;
+
+/**
+ * Notification type enumeration model.
```

Comment:
The enum name tells exactly what this doc tells, remove it (also check in all other classes)